engines/director/lingo/lingo.h

```diff
@@ -291,6 +291,15 @@ struct LingoArchive {
 	void addNamesV4(Common::SeekableReadStreamEndian &stream);
 };
 
+struct LingoState {
```

Comment:
Please add some docs here, e.g. purpose of this struct and fields descriptions

Reply:
Done